ReactWindows/ReactNative/Views/Web/ReactWebViewManager.cs

```diff
@@ -76,6 +77,24 @@ public override JObject ViewCommandsMap
             }
         }
 
+        /// <summary>
+        /// Sets the background color for the <see cref="WebView"/>.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="color">The masked color value.</param>
+        [ReactProp(ViewProps.BackgroundColor, CustomType = "Color")]
+        public void SetBackgroundColor(WebView view, uint? color)
+        {
+            if (color.HasValue) {
+                var parsedColor = ColorHelpers.Parse(color.Value);
+                // "style={{ backgroundColor: 'transparent' }}" gives #00000000,
+                // but we need #00FFFFFF to make the background color transparent
+                view.DefaultBackgroundColor = parsedColor.A == 0 ? Colors.Transparent : parsedColor;
```

Comment:
I'm curious why this is inconsistent with the background color in ReactViewManager. Why do we only need to apply this change here?

Reply:
Hmm, right. Though I think this modification should be applied all the places where `react-native-windows` uses colors, this PR should be simple. So, I will remove this modification.